aspnetcore/fundamentals/minimal-apis/7.0-samples/todo-group/Program.cs

```diff
@@ -25,14 +25,14 @@
 app.MapGet("/", () => "Hello World!");
 
 // todo endpoints
-var todos = app.MapGroup("/todos").WithTags("Todo Endpoints").AddRouteHandlerFilter(async (context, next) =>
+var todos = app.MapGroup("/todos").WithTags("Todo Endpoints").AddEndpointFilter(async (context, next) =>
```

Comment:
@fiyazbinhasan I used the MapGroup method here

Reply:
Got it! With a bit of refactoring all of these routes could be abstracted in the `RouteHandlers` class,

```
public static RouteGroupBuilder MapTodosApi(this RouteGroupBuilder group)
    {
        group.MapGet("/", GetAllTodos);
        group.MapGet("/{id}", GetTodo);
        group.MapPost("/", CreateTodo);
        group.MapPut("/{id}", UpdateTodo);
        return group;
    }
```

In program.cs you will only have to write this,

```
app.MapGroup("/todos").MapTodosApi();
```

What do you think? 😸

Reply:
@fiyazbinhasan Seems I'm mistaken. Let me adjust it to use the static method for that rather than create mine.

Reply:
Sure you're right. Let me do that. Thanks for the update.